pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -461,6 +467,15 @@ The delayed message index bucket time step(in seconds) in per bucket snapshot se
         )
     private int zooKeeperCacheExpirySeconds = -1;
 
+    @Deprecated
```

Comment:
Why are we add deprecated fields? Seems not necessary.

Reply:
If the user uses the old properties, it just add `zookeeperAllowReadOnlyOperations` config.

Reply:
Could we remove the `zooKeeperAllowReadOnlyOperations` config?  Because there seems is no compatible issue

Reply:
Seems we need to remove it.